requirements-dev.in

```
@@ -17,10 +17,14 @@ pytest-xdist==3.5.0
 
 ### aiobotocore dev dependencies
 
-dill~=0.3.3  # Requirement for tests/test_patches.py
+dill~=0.3.3
 docker~=7.1
+docutils~=0.16
 moto[server,s3,sqs,awslambda,dynamodb,cloudformation,sns,batch,ec2,rds]~=4.2.9
+pip~=24.3.1
 pre-commit~=3.5.0
 pytest-asyncio~=0.23.8
+requests~=2.32.3
 time-machine~=2.15.0
-tomli; python_version < "3.11"  # Requirement for tests/test_version.py
+tomli; python_version < "3.11"
+werkzeug~=3.0.6
```

this should be pulled in via moto[server] IIRC

---

May well be, but we use `werkzeug` directly it in our test suite and if moto were to drop that dependency, it would break us. Therefore I believe we should make that dependency of ours explicitly.

---

sorry afk but the only reason we use it directly is because moto uses it I think no? so if they changed it we'd have to change our code anyways

---

It's a direct dep nevertheless:
```suggestion
werkzeug~=3.0.6  # Used in moto_server.py
```